src/components/StakingVaults/ProviderCard.tsx

```diff
@@ -95,15 +96,16 @@ export const ProviderCard: React.FC<ProviderCardProps> = ({
 
         return (
           lp.includes(e.assetId as OpportunityId) &&
-          walletSupportsChain({
-            chainId: e.chainId,
-            wallet,
-            isSnapInstalled,
-            checkConnectedAccountIds: chainAccountIds,
-          })
+          (!isConnected ||
```

Comment:
We don't want to narrow opportunities by wallet support since... there is no wallet here.